### PR TITLE
Added GetMembersOfGroup

### PR DESCRIPTION
**Pull request by 5ARMALE**
Added support for GetMembersOfGroup in order to be able to list and get info about Clan members.

### CODE REVIEW

File: pydest/api.py [hunk 1]
```diff
@@ -295,6 +295,21 @@ def __init__(self, api_key, session):
...
+            group_id (int):
+                Destiny group ID
+        
+        Returns:
+            json(dict)
+        """
+        # /{group_id}/Members/?currentPage=1
+        url = GROUP_URL + '{}/Members/?currentPage=1'
+        url = url.format(group_id)
+        return await self._get_request(url)
```
Comment (jgayfer) [inline pydest/api.py]: Can we add an integration test for this, please?

Comment (jgayfer) [reply pydest/api.py]: This is where they would go! :heart:
https://github.com/jgayfer/pydest/blob/master/pydest/test/test_integration.py

**Comment by 5ARMALE** [comment]
Whoa! Just forgot to add it! Now it should be all right.

Thanks!

File: pydest/api.py (hunk 1 above)
Comment (5ARMALE) [reply pydest/api.py]: Whoa! Just forgot to add it! Now it should be all right.

Thanks!

File: pydest/api.py [hunk 2]
```diff
@@ -295,6 +295,21 @@ def __init__(self, api_key, session):
...
+        
+        Args:
+            group_id (int):
+                Destiny group ID
+        
+        Returns:
+            json(dict)
+        """
+        # /{group_id}/Members/?currentPage=1
+        url = GROUP_URL + '{}/Members/?currentPage=1'
```
Comment (jgayfer) [inline pydest/api.py]: I think ideally we could specify which page we want (if any). Perhaps we add an additional optional param to the function?

```
 async def get_members_of_group(self, group_id, page=1):
    url = GROUP_URL + '{}/Members/?currentPage={}'
    url = url.format(group_id, page)
```

The default doesn't necessarily need to be `1` either; the default could be to omit the page param, but my guess would be that by default they would show the first page.

Comment (5ARMALE) [reply pydest/api.py]: Added! I just did some tests without passing the page path parameter and it returns the first page by default as you say. Anyway clans seem to be limited to 100 people, and it lists 100 entries per page, so the first page should be enough *gasp*.

Thanks!

Comment (jgayfer) [reply pydest/api.py]: Sounds good!

File: pydest/api.py [hunk 3]
```diff
@@ -295,6 +295,21 @@ def __init__(self, api_key, session):
...
+        """Gets members of a group_id 
+        
+        Args:
+            group_id (int):
+                Destiny group ID
+        
+        Returns:
+            json(dict)
+        """
+        # /{group_id}/Members/?currentPage={page}
```
[the remark this answers is not in the document]
Comment (5ARMALE) [reply pydest/api.py]: It should be OK, right? I just put it there to be "loyal" to the rest of the methods.

[the remark this answers is not in the document]
Comment (jgayfer) [reply pydest/api.py]: Ah yes, I forgot we added those in. I might clean up later. Thanks!

**Comment by jgayfer** [comment]
Released as part of version `0.3.0` - https://pypi.org/project/pydest/0.3.0/

**Pull request merged**